docker_cli/cli/build_commands.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+from typing import List
+
+from ..commands import clone
+from ._utils import add_tag_argument, help_formatter
+
+
+def init_build_parsers(subparsers: argparse._SubParsersAction) -> None:
+    """
+    Augment an argument parser with build commands.
```

Comment:
Can we clarify what we mean by "build commands". Ideally, an arbitrary person on the isce3 development team could read this docstring and get a reasonable understanding of what's going on.